core/controller/src/main/scala/whisk/core/entitlement/ActivationThrottler.scala

```diff
@@ -73,7 +73,7 @@ class ActivationThrottler(consulServer: String, loadBalancer: LoadBalancer, conc
         // Any sort of partitioning will be ok for monitoring
         Future.sequence(userActivationCounter.groupBy(_._1.take(1)).map {
             case (prefix, items) =>
-                val key = ControllerKeys.userActivationCountKey + "/" + prefix
+                val key = ControllerKeys.userActivationCountKey(controllerInstance) + "/" + prefix
```

Comment:
Should this (the throttle) aggregate over all instances - a comment is warranted in this class that explains the accounting is local to an instance (and hence clears on restarts).